data/mods/Magiclysm/monsters/monsters.json

```diff
@@ -359,5 +359,39 @@
     "death_function": [ "NORMAL" ],
     "regenerates": 1,
     "flags": [ "SEES", "SMELLS", "HEARS", "STUMBLES", "PATH_AVOID_FIRE", "REVIVES" ]
+  },
+  {
+    "id": "mon_loup_garou",
```

Comment:
Maybe it should say `mon_werewolf` instead of `mon_loup_garou` to be easier to understand for future contributors. Keep loup garou in the name but change the id.

Reply:
I was originally thinking that I'd add multiple types of werewolves in this update but I'm thinking I'm behind on getting this finished so I'll probably go ahead and change the name to werewolf and then if I add more types of werewolf later I can update or even just name them `mon_werewolf_1` and so on.

Reply:
Well maybe it's ok then,the description is pretty clear and you don't need to understand the id to use it.